stable/kubernetes-dashboard/README.md

```diff
@@ -32,6 +32,18 @@ helm delete my-release
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Upgrading an existing Release to a new major version
+
+A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an
+incompatible breaking change needing manual actions.
+
+### To 2.0.0
+
+This version upgrades to kubernetes-dashboard v2.0.0 along with changes in RBAC management: all secrets are explicitely created and ServiceAccount do not have permission to create any secret.
+
+In order to upgrade, either remove the secrets or uninstall/reinstall.
```

Comment:
I believe the label updates require an uninstall/reinstall of the release, because the deployment's label selector is immutable. The upgrade instructions should probably just say a `helm del --purge` is required

Reply:
Thanks, I forgot to update the README.

Reply:
Fixed.